components/pedigree/resources/src/main/resources/pedigree/model/import.js

```diff
@@ -1561,7 +1560,7 @@ define([
                                    properties["comments"] = "(untested)\n" + properties["comments"];
                                }
                                break;
-                           case "O":
+                           case "C":
```

Comment:
"O" is already a case in this switch statement, and "C" seems like the best fit based on the documentation.

Reply:
looks correct